src/fees/BuilderFeeManager.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import { IBuilderFeeManager } from "./interfaces/IBuilderFeeManager.sol";
+import { Ownable } from "../lib/utils/Ownable.sol";
+
+contract BuilderFeeManager is Ownable, IBuilderFeeManager {
+    mapping(address => uint256) private feeOverride;
+    uint256 private defaultFeeBPS;
+
+    event FeeOverrideSet(address indexed, uint256 indexed);
+    event DefaultFeeSet(uint256 indexed);
+
+    constructor(uint256 _defaultFeeBPS, address feeManagerAdmin) {
+        defaultFeeBPS = _defaultFeeBPS;
+        _transferOwnership(feeManagerAdmin);
+    }
+
+    function setDefaultFee(uint256 amountBPS) external onlyOwner {
+        require(amountBPS < 2001, "Fee too high (not greater than 20%)");
+        defaultFeeBPS = amountBPS;
+        emit DefaultFeeSet(amountBPS);
+    }
+
+    function setFeeOverride(address tokenContract, uint256 amountBPS) external onlyOwner {
+        require(amountBPS < 2001, "Fee too high (not greater than 20%)");
+        feeOverride[tokenContract] = amountBPS;
```

Comment:
what's the use case for this? in our experience fee overrides were never really used.

Reply:
copied for original zora impl so wasn't sure if you guys had a good reason for it or not. just removed it as I don't think it will be necessary.